content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -377,21 +375,27 @@ configMapGenerator:
 EOF
 ```
 
-Apply the kustomization directory to create the ConfigMap object.
+Apply the kustomization directory to create the ConfigMap:
```

Comment:
Should ConfigMap use code formatting?

Reply:
Probably - would it be OK to save that update for a follow-up PR?

Reply:
👍